From 58a3749bc5c2060968949473d59502a5d0973f6d Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Thu, 20 Sep 2018 15:24:35 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399 fix node unit name coding style

Fix node unit name coding style, such as unit name vs reg, or
unit name vs format.

Change-Id: I8a3e28cc1949acc23622a4a1a20e4ab1479bf512
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399.dtsi | 8 +++++---
 1 file changed, 5 insertions(+), 3 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399.dtsi b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
index 0e3d7c9da3ee..815a8c131239 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399.dtsi
@@ -403,7 +403,7 @@
 		status = "disabled";
 	};
 
-	usbdrd3_0: usb@fe800000 {
+	usbdrd3_0: usb0 {
 		compatible = "rockchip,rk3399-dwc3";
 		clocks = <&cru SCLK_USB3OTG0_REF>, <&cru SCLK_USB3OTG0_SUSPEND>,
 			 <&cru ACLK_USB3OTG0>, <&cru ACLK_USB3_GRF>;
@@ -416,6 +416,7 @@
 		#size-cells = <2>;
 		ranges;
 		status = "disabled";
+
 		usbdrd_dwc3_0: dwc3@fe800000 {
 			compatible = "snps,dwc3";
 			reg = <0x0 0xfe800000 0x0 0x100000>;
@@ -435,7 +436,7 @@
 		};
 	};
 
-	usbdrd3_1: usb@fe900000 {
+	usbdrd3_1: usb1 {
 		compatible = "rockchip,rk3399-dwc3";
 		clocks = <&cru SCLK_USB3OTG1_REF>, <&cru SCLK_USB3OTG1_SUSPEND>,
 			 <&cru ACLK_USB3OTG1>, <&cru ACLK_USB3_GRF>;
@@ -448,6 +449,7 @@
 		#size-cells = <2>;
 		ranges;
 		status = "disabled";
+
 		usbdrd_dwc3_1: dwc3@fe900000 {
 			compatible = "snps,dwc3";
 			reg = <0x0 0xfe900000 0x0 0x100000>;
@@ -2075,7 +2077,7 @@
 		};
 	};
 
-	mipi_dphy_tx1rx1: mipi-dphy-tx1rx1@0xff968000 {
+	mipi_dphy_tx1rx1: mipi-dphy-tx1rx1@ff968000 {
 		compatible = "rockchip,rk3399-mipi-dphy";
 		reg = <0x0 0xff968000 0x0 0x8000>;
 		clocks = <&cru SCLK_MIPIDPHY_REF>,
-- 
2.35.3

